### ISSUE TITLE
Source code security inspection

### ISSUE THREAD

**Issue by megai2** [megai2/d912pxy#11]
Hello there, need a man to check sources and make a statement on it when they are released.

**Comment by handsomematt** [comment]
ok, email me matt@handsomematt.co.uk

**Comment by megai2** [comment]
Released source code. If you up to make some inspections, please post your results here.

**Comment by Matthew-Garvey** [comment]
I've been looking through this on IDA, but I wouldn't mind a look at the source so I could give you a vouch since I'm more of a graphics programmer and not a security expert.

Email is on my profile.

**Comment by Matthew-Garvey** [comment]
Is there any particular reason you didn't include the P7logger source code in its directory, and instead asked to have it downloaded from its origin website? 

I could theoretically download it and it would work fine and without suspicious activity, but there's no guarantee that what you included in the prebuilt binaries is exactly the same thing. Providing the full source code as you built from is the only way I'd be willing to vouch for this.

**Comment by megai2** [comment]
For P7logger as @handsomematt stated, this is a common pratice and even if i include sources, it will not give you something about gaurantee TBH, only make you compile it out of box. Any way, fine for me to add them.

For D3D9Proxy, project not uses original sources from /extern, instead that sources are heavily modified and incorporated in project, i made that dir to point out that i have used it.

**Comment by Matthew-Garvey** [comment]
Appreciated. I figured that was the case for the D3D9Proxy source, but I wanted to make sure.

Of course, it's no guarantee about the binaries that you have pre-compiled, but I am satisfied with what is provided that all logging capabilities are for the client. I can attempt a more detailed analysis of the files in IDA at another time. From what I saw, things are perfectly fine.

It's clear that you've put a lot of effort into this, and it's certainly no trivial task. I'm sorry you got caught up in the reddit shit-storm.

That being said, simple client logging is a _very_ small task, and including a logging library with remote capabilities is just asking for people to get on your case. You will forever have people scrutinizing your releases if you continue using P7 without disclosing your source code.
